src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +80,36 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
-			year, month, events, minDate, containerId, $container;
+		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ), // Ability to extend the calendar
+			year, month, events, minDate, maxDate, containerId, $container,
+			currDate = new Date(),
+			currYear = currDate.getFullYear(),
+			currMonth = currDate.getMonth();
 
 		events = getEvents( $elm );
 		containerId = $elm.data( "calevtSrc" );
 		$container = $( "#" + containerId ).addClass( componentName + "-cal" );
 
 		minDate = events.minDate;
-		year = settings.year || minDate.getFullYear();
-		month = settings.month || minDate.getMonth();
+		maxDate = events.maxDate;
+
+		if ( minDate.getTime() > currDate.getTime() ) {
```

Comment:
In case the lower event is something like 2013, we want the calendar to be set as the date of today. Presently the fonction getEvents does do this. But I agree, the fonction should return it instead of (re)modifying. What do you think?